import java.lang.*;

//When we run this application it generates exception
class Exception_Demo1
{

	public static void main(String args[])
	{
			// there are 4 elements in array.
		int arr[]={10,20,30,40};	
		

		System.out.println("Inside main"+arr[6]);
	}
}


